src/main/java/uk/gov/digital/ho/hocs/casework/migration/MigrationCaseDataService.java

```diff
@@ -87,7 +89,7 @@ public void updateCaseData(UUID caseUUID, UUID stageUUID, Map<String, String> da
         if (data != null) {
             log.debug("Data size {}", data.size());
             CaseData caseData = getCase(caseUUID);
-            caseData.update(data, objectMapper);
+            //caseData.update(data, objectMapper);
```

Comment:
This just wants deleting at this point, outside of the scope of this PR, but happy to raise at the same time